front/lib/auth.ts

```diff
@@ -129,13 +129,13 @@ export class Authenticator {
     if (user && workspace) {
       [role, subscription, flags] = await Promise.all([
         (async (): Promise<RoleType> => {
-          const membership = await Membership.findOne({
-            where: {
-              userId: user.id,
-              workspaceId: workspace.id,
-            },
+          const memberships = await MembershipResource.getActiveMemberships({
```

Comment:
I would be in favour of having a `getActiveMenbership` function to avoid the [0] deref. We'll likely need it in other places.

Reply:
done